crates/sui-core/src/validator_tx_finalizer.rs

```diff
@@ -100,15 +87,51 @@ impl ValidatorTxFinalizerMetrics {
     }
 }
 
+pub struct ValidatorTxFinalizerConfig {
+    pub tx_finalization_delay: Duration,
+    pub tx_finalization_timeout: Duration,
+    /// Incremental delay for validators to wake up to finalize a transaction.
+    pub validator_delay_increments_sec: u64,
+    pub validator_max_delay: Duration,
+}
+
+#[cfg(not(any(msim, test)))]
+impl Default for ValidatorTxFinalizerConfig {
+    fn default() -> Self {
+        Self {
+            // Only wake up the transaction finalization task for a given transaction
+            // after 1 mins of seeing it. This gives plenty of time for the transaction
+            // to become final in the normal way. We also don't want this delay to be too long
+            // to reduce memory usage held up by the finalizer threads.
+            tx_finalization_delay: Duration::from_secs(60),
+            // If a transaction can not be finalized within 1 min of being woken up, give up.
+            tx_finalization_timeout: Duration::from_secs(60),
+            validator_delay_increments_sec: 10,
+            validator_max_delay: Duration::from_secs(180),
+        }
+    }
+}
+
+#[cfg(any(msim, test))]
```

Comment:
I think this gets applied to simtests in other crates, but not regular tests. Maybe better to verify.

Reply:
Yes and that's fine. It's mostly to make sure the tests within the finalizer use these config.